stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -713,6 +713,9 @@ for f in (:exp, :cis, :log, :sqrt,
     @eval $f(D::Diagonal) = Diagonal($f.(D.diag))
 end
 
+# Cube root of a real-valued diagonal matrix
+cbrt(A::Diagonal{T}) where {T<:Real} = Diagonal(cbrt.(A.diag))
```

Comment:
```suggestion
cbrt(A::Diagonal{<:Real}) = Diagonal(cbrt.(A.diag))
```
It's better style to put the type parameter directly with the argument unless you are going to actually use it in the body of the function.  Similarly elsewhere.